run/helm/snap/README.md

```diff
@@ -0,0 +1,233 @@
+# Snap
+<img src="https://cloud.githubusercontent.com/assets/1744971/20331694/e07e9148-ab5b-11e6-856a-e4e956540077.png" width="70%">
+
+[Snap](https://github.com/intelsdi-x/snap) is a powerful open source telemetry framework. It allows to collect, process and publish system data gathered from various environments. Deployment of Snap in Kubernetes cluster gives a possibilty to monitor pods and nodes in the cluster. It can collect metrics i.e. from host and other containers. To learn more please go to [Snap documentation](https://github.com/intelsdi-x/snap/blob/master/README.md) and [Snap integrations repo](https://github.com/intelsdi-x/snap-integration-kubernetes).
+
+1. [QuickStart](#1-quick-start)
+2. [Introduction](#2-introduction)
+3. [Prerequisites](#3-prerequisites)
+4. [Installing the Chart](#4-installing-the-chart)
+5. [Uninstalling the Chart](#5-uninstalling-the-chart)
+6. [Configuration](#6-configuration)
+7. [Loading Snap plugins](#7-loading-snap-plugins)
+
+## 1. Quick Start
+
+`$ helm install stable/snap`
+
+## 2. Introduction
+
+This chart bootstraps Snap on all nodes in your Kubernetes Cluster via DaemonSet using Helm package manager. It uses official Snap for Kubernetes Docker image [intelsdi/snap4kube](https://hub.docker.com/r/intelsdi/snap4kube/).
+
+As you may read in [Snap documentation](https://github.com/intelsdi-x/snap#overview) the framework consists of two main components: `snapteld` daemon and plugins. This chart deploys and evokes only Snap daemon `snapteld`. In order to begin collection of any metrics we need some plugins. It is up to the user which plugin they wish to use. The list of all plugins is available in [Plugin Catalog](https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_CATALOG.md). The user needs to download and load plugins. More information on how to configure and load Snap plugins can be found in [Loading Snap plugins](#7-loading-snap-plugins) section of this document as well as in Snap integration repo in section [Running Snap in Kubernetes](https://github.com/intelsdi-x/snap-integration-kubernetes/tree/master/run/kubernetes).
+
+## 3. Prerequisites
+
+* Kubernetes 1.4+ with Beta APIs enabled
+
+## 4. Installing the Chart
+
+To install the chart with the release name my-snap use command:
+
+`$ helm install --name my-snap stable/snap`
+
+This command deploys Snap on the Kubernetes cluster in the default configuration. To learn how to override the default configuration see section [Configuration](#6-configuration).
+
+## 5. Uninstalling the Chart
+
+To uninstall/delete the my-snap daemonset use command:
+
+`$ helm delete my-snap --purge`
```

Comment:
I would not recommend the use of `--purge` in basic documentation, it is more of an "advanced" usage flag